operator/v1/types_ingress.go

```diff
@@ -1525,16 +1525,17 @@ type IngressControllerTuningOptions struct {
 	// A zero value for reloadInterval tells the IngressController to choose the default,
 	// which is currently 5s and subject to change without notice.
 	//
-	// This field expects an unsigned duration string of decimal numbers, each with optional
-	// fraction and a unit suffix, e.g. "100s", "1m30s". Valid time units are "s" and "m".
+	// Expects an unsigned duration string of decimal numbers, each with optional
+	// fraction and a unit suffix, eg "300ms", "1.5h" or "2h45m".
+	// Valid time units are "ns", "us" (or "µs" U+00B5 or "μs" U+03BC), "ms", "s", "m", "h".
 	//
 	// Note: Setting a value significantly larger than the default of 5s can cause latency
 	// in observing updates to routes and their endpoints. HAProxy's configuration will
 	// be reloaded less frequently, and newly created routes will not be served until the
 	// subsequent reload.
 	//
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(s|m))+)$
+	// +kubebuilder:validation:Pattern=^(0|([0-9]+(\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+)$
```

Comment:
Why is "µs" included twice? ...Oh, we include it twice in the validation for `spec.tuningOptions.healthCheckInterval` too.

Reply:
Oh, I should have read the comment above.  I see now:  Those are different code points.  (In my defense, I did a copy and paste and ctrl-f, but it turns out both Firefox and Emacs treat them the same after character folding.)

Reply:
Yea I just copied and pasted in vim, I suppose that should be good enough.